pandas/tests/test_groupby.py

```diff
@@ -6279,6 +6279,22 @@ def test_func(x):
         expected = DataFrame()
         tm.assert_frame_equal(result, expected)
 
+    def test_groupby_apply_none_first(self):
+        # GH 12824. Tests if apply returns None first.
+        test_df = DataFrame({'groups': [0, 1, 1, 1],
+                             'vars': [8, 7, 4, 5]})
```

Comment:
add both test cases from the issue (1 already works, 1 you are fixing), so that they are in the same place.